docs/src/conf.py

```diff
@@ -16,6 +17,15 @@
 source_code_root = (Path(__file__).parents[2]).absolute()
 sys.path.append(str(source_code_root))
 
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
+rtd_version = os.environ.get("READTHEDOCS_VERSION", "latest")
```

Comment:
It seems a little inconsistent having this line outside the `if on_rtd:` logic while `rtd_project` is set within given that both of these are only used within that scope.

Reply:
True. For us, i.e. in ESMValTool, we do this because there are a few links between projects where we also want to link to the latest rtd_version when the documentation is built locally. But perhaps this gets in the way here?

Reply:
I suppose that makes sense if it's following a pattern. With that said, I can't imagine a case where we would add another project in this code so I think it ought to be fine to change it.

Reply:
Done.